include/dem/particle_wall_contact_force.h

```diff
@@ -1143,10 +1144,88 @@ class ParticleWallContactForce
       }
   }
 
+  /**
+   * @brief Set every containers needed to carry the heat transfer rate
+   * calculation.
+   *
+   * @param[in] dem_parameters DEM parameters declared in the .prm
+   * file.
+   */
+  void
+  set_multiphysic_properties(const DEMSolverParameters<dim> &dem_parameters)
+  {
```

Comment:
I would move the implementation of that function to the  .cc. No need in having so much stuff defined in .h :)